packages/compose/src/hooks/use-reduced-motion/index.js

```diff
@@ -8,7 +8,9 @@ import useMediaQuery from '../use-media-query';
  *
  * @type {boolean}
  */
-const IS_IE = window.navigator.userAgent.indexOf( 'Trident' ) >= 0;
+const IS_IE =
+	typeof window !== 'undefined' &&
```

Comment:
@koke - this is another place where we could take advantage of some sort of platform detection. It would be useful not only for detecting native iOS or Android.

Reply:
I think in this case we could have a separate native implementation, since react native 0.60 exposes those APIs in [AccessibilityInfo](https://facebook.github.io/react-native/docs/accessibilityinfo.html)

I can't test this on 0.60 right now, but something like this should do it:

```js
function useReducedMotion() {
    const [ reducedMotion, setReducedMotion ] = useState( false );
    useEffect( () => {
        const updateReducedMotion = () => AccessibilityInfo.isReduceMotionEnabled().then( setReducedMotion );
        updateReducedMotion();
        AccessibilityInfo.addEventListener('reduceMotionChanged', updateReducedMotion);
        return () => {
            AccessibilityInfo.removeEventListener('reduceMotionChanged', updateReducedMotion);
        };
    } );
    
    return reducedMotion;
}
```